### DISCUSSION TITLE
Game bot to do automation

### DISCUSSION THREAD

**Discussion by dollarotieno** [community/community#66415] [Mobile] [answered]
### Select Topic Area

General

### Body

I have a game app and I need is to be able to swap invite links out manually to bot and for bot to run a course of 50 invites through the method video I am sending. 
I also need to be able to set it to run on a schedule on its own that I can set for either 1hr or over night if need be. 
You will have to show me as well as write out how to operate bot please. I have some videos I can also share. I have been trying to get this method to be able to automated and its not so basically I need the bot to be for automation of the process in video but I need to be able to swap out the monoply go player link . Also I have windows 11 pro don't know if that matters or not. For the phone swap step I would actually do it as 
Samsung S21 ultra 
Samsung s22 ultra 
And the Xiaomi 11Tpro 
I have found that switching between these 3 alternating work best manually so it should work the same for you. 
I just keep rotating which one 
https://drive.google.com/drive/folders/1BPZNO0YN6DTfmKukDuRVIRb-uJ7AVi6X
Play MONOPOLY GO! with me! Download it here: https://s.scope.ly/hkhXX1nlPWQ
https://s.scope.ly/hkhXX1nlPWQ

**Comment by stevepopovich** [comment, marked as answer]
Hi @dollarotieno This forum is for feedback about the GitHub Mobile Native app. If you have feedback for GitHub, that'd be great. Otherwise, I think you will have more luck with a site like StackOverflow.

**Comment by Yashu690** [comment]
New id aviator predictor bot
